.storybook/main.js

```diff
@@ -64,6 +64,13 @@ module.exports = {
       }),
     );
 
+    config.resolve.fallback = {
+      ...config.resolve.fallback,
+      fs: false,
+      stream: false,
+      zlib: false,
+    };
```

Comment:
This is needed because the UGC form has an entire page story, so it will also import the Context logic that houses the client-side fetch.